### PR TITLE
Qt5 support for CalcQt example

### PR DESCRIPTION
**Pull request by konserw**
Now CalcQt example should work on both Qt4 and Qt5.
I think this also fixes issue #83 as i've changed calcqt library to libcalcqt.

### CODE REVIEW

**Comment by konserw** [comment]
merge

File: examples/CalcQt/CMakeLists.txt [hunk 1]
```diff
@@ -1,21 +1,49 @@
 project(CalcQt)
 
-find_package(Qt4 COMPONENTS QtCore QtGui QtTest)
+set(CALCQT_HEADERS src/CalculatorWidget.h)
+set(CALCQT_SOURCES src/CalcQt.cpp src/CalculatorWidget.cpp)
+set(LIBCALCQT_BUILD OFF)
```
Comment (paoloambrosio) [inline examples/CalcQt/CMakeLists.txt]: Is `LIBCALCQT_BUILD` really needed? If it's for the `if(LIBCALCQT_BUILD)` later on, can't you use the `QT_LIBRARIES` variable?

**Comment by paoloambrosio** [comment]
Nice PR! I trust you have tested it with QT5, and I have with QT4. Apart from the comment it should be ready to merge.

File: examples/CalcQt/CMakeLists.txt (hunk 1 above)
Comment (konserw) [inline examples/CalcQt/CMakeLists.txt]: Good idea ;) Let's check that.

**Comment by konserw** [comment]
Ok, now it should be ready for merging if travis tests passes.

**Pull request merged**